tests/system/feature_store/test_feature_store.py

```diff
@@ -176,6 +177,32 @@ def test_ingest_and_query(self):
         self._logger.debug("Get online feature vector")
         self._get_online_features(features, features_size)
 
+    def test_backwards_compatibility_step_vs_state(self):
```

Comment:
I don't see a reason for this to be a system test - you just create an object - transform to dict - and create the object again from dict - can be a unit test no ?

Reply:
Moved to unit-tests.